firmware/application/apps/ui_playlist.cpp

```diff
@@ -219,7 +258,7 @@ void PlaylistView::handle_replay_thread_done(const uint32_t return_code) {
         stop(true);
     } else if (return_code == ReplayThread::READ_ERROR) {
         stop(false);
-        file_error();
+        file_error("Illegal grammar(255)");
```

Comment:
See, line number no longer matches. You can make it work if you really want it with the __LINE__ macro. But it’s not really necessary.

Reply:
Thank you very much! Since it's just for debug, I think we could safely remove the line number now.

Reply:
fixed. Thank you!